src/prettytable/prettytable.py

```diff
@@ -96,6 +96,8 @@ class PrettyTable:
     _xhtml: bool
     _align: dict[str, AlignType]
     _valign: dict[str, VAlignType]
+    _max_width: dict[str, int]
+    _min_width: dict[str, int]
```

Comment:
```suggestion
    _min_width: dict[str, int]
    _max_width: dict[str, int]
```
Same order as the following pair.